library/src/otx/types/label.py

```diff
@@ -63,7 +74,7 @@ def from_num_classes(cls, num_classes: int) -> LabelInfo:
         )
 
     @classmethod
-    def from_dm_label_groups(cls, dm_label_categories: LabelCategories) -> LabelInfo:
+    def from_dm_label_groups(cls, dm_label_categories: HierarchicalLabelCategories) -> LabelInfo:
```

Comment:
Since we deleted this in OTXDataset and we define LabelInfo directly, could you clean this class? Specifically, we don't need "from_dm_label_groups", "from_dm_label_groups_arrow", etc.